src/trading/getEthFlowTransaction.ts

```
@@ -3,7 +3,7 @@ import { LimitTradeParametersFromQuote, TransactionParams } from './types'
 import { calculateUniqueOrderId, EthFlowOrderExistsCallback } from './calculateUniqueOrderId'
 import { getOrderToSign } from './getOrderToSign'
 import { type EthFlow, EthFlow__factory } from '../common/generated'
-import { BARN_ETH_FLOW_ADDRESSES, CowEnv, ETH_FLOW_ADDRESSES, SupportedChainId } from '../common'
+import { BARN_ETH_FLOW_ADDRESS, CowEnv, ETH_FLOW_ADDRESS, SupportedChainId, WRAPPED_NATIVE_CURRENCIES } from '../common'
```

ESLint

[error] 'WRAPPED_NATIVE_CURRENCIES' is defined but never used.

(unused-imports/no-unused-imports)